test/stress/mt_batching_test.go

```diff
@@ -0,0 +1,119 @@
+package test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/weaviate/weaviate-go-client/v4/test/testsuit"
+	"github.com/weaviate/weaviate-go-client/v4/weaviate/filters"
+	"github.com/weaviate/weaviate-go-client/v4/weaviate/graphql"
+	"github.com/weaviate/weaviate/entities/models"
+)
+
+func TestMTBatching_stress(t *testing.T) {
+	className := "Collection"
+	client := testsuit.CreateTestClient(true)
+	ctx := context.Background()
+
+	defer func() {
+		err := client.Schema().ClassDeleter().WithClassName(className).Do(ctx)
+		require.Nil(t, err)
+	}()
+
+	t.Run("Create multi-tenancy collection", func(t *testing.T) {
+		err := client.Schema().ClassCreator().WithClass(&models.Class{
+			Class: className,
+			Properties: []*models.Property{
+				{
+					DataType: []string{"string"},
+					Name:     "name",
+				},
+				{
+					DataType: []string{"int"},
+					Name:     "count",
+				},
+				{
+					DataType: []string{"number"},
+					Name:     "price",
+				},
+			},
+			MultiTenancyConfig: &models.MultiTenancyConfig{
+				AutoTenantActivation: true,
+				AutoTenantCreation:   true,
+				Enabled:              true,
+			},
+			Vectorizer: "none",
+		}).Do(ctx)
+		if err != nil {
+			t.Fatalf("Create collection: %v", err)
+		}
+	})
+
+	vector := make([]float32, 16)
+	for i := range vector {
+		vector[i] = float32(i) / 16
+	}
```

Comment:
not really an issue in this case, but FYI: If you always use the same vector inserts and searches are very slow

Reply:
Ah, really? Makes sense really and I'll change anyway

Reply:
you will have a ton of warnings in your weaviate log if you do this😆